packages/react-on-rails-pro/tests/registerServerComponent.client.test.jsx

```
@@ -9,10 +9,10 @@ import { screen, act } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import * as path from 'path';
 import * as fs from 'fs';
-import { createNodeReadableStream, getNodeVersion } from './testUtils.js';
-import ReactOnRails from '../src/ReactOnRails.client.ts';
-import registerServerComponent from '../src/pro/registerServerComponent/client.tsx';
-import { clear as clearComponentRegistry } from '../src/pro/ComponentRegistry.ts';
+import { createNodeReadableStream, getNodeVersion } from '../../react-on-rails/tests/testUtils.js';
```

ESLint

[error] Relative import from another package is not allowed. Use `react-on-rails/tests/testUtils.js` instead of `../../react-on-rails/tests/testUtils.js`

(import/no-relative-packages)